.gitignore

```diff
@@ -1,3 +1,5 @@
 /book/
 target/
 po/messages.pot
+.DS_Store
+po/pt-BR.mo
```

Comment:
Let's move this line up to the other `po/` line and change it to `po/*.mo` instead!